src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
+        Color color = ColorSupplier.getRandomColor();
+        int number = random.nextInt(100) + 1;
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)